src/main/java/com/spartronics4915/frc2020/subsystems/Launcher.java

```diff
@@ -0,0 +1,157 @@
+    }
+    /**
+     * @param angle Angle in degrees above horizontal you want the angle adjuster to go to
+     */
+    public void setAngle(double angle) {
+        //sets target angle to given angle
+        targetAngle=angle;
+    }
+
+    /**
+     * @param rpm RPM you want the flywheel to target
+     */
+    public void setRPM(double rpm) {
+        //sets target rpm for flywheel to given rpm
+        targetRPM=rpm;
+    }
+    /**
+     * @return Angle in degrees above horizontal that the angle adjuster is targeting
+     */
+    public double getTargetAngle() {
+        //returns current target angle of angle adjuster
+        return targetAngle;
+    }
+    /**
+     * @return RPM that the flywheel is targeting
+     */
+    public double getTargetRPM() {
+        //returns current target RPM of shooter
+        return targetRPM;
+    }
+    /**
+     * @return Current angle in degrees above horizontal of the angle adjuster
+     */
+    public double getCurrentAngle() {
+        //returns current angle of angle adjuster
+        //NEED ENC OR POT
+        mAngleAdjusterEncoder.getPosition();
+        return 0.0;
+    }
+    /**
+     * @return The current RPM of the flywheel
+     */
+    public double getCurrentRPM() {
+        //returns current RPM of shooter
+        return mFlywheelEncoder.getVelocity();
+    }
+
+    /**
+     * @param distance Horizontal distance in meters from the shooter to the target
+     * @return The angle in degrees above horizontal that is calculated to be necessary to hit the target based off of the input distance
+     */
+    public double calcAngle(double distance) {
+        //computes and returns angle for angle adjuster based on input distance
+        double angle=0.0;
+        return angle;
+    }
+
+    /**
+     * @param distance Horizontal distance in meters from the shooter to the target
+     * @return RPM calculated to be necessary to hit the target based of of the input distance
+     */
+    public double calcRPM(double distance) {
+        //computes and returns RPM based on input distance
+        double RPM=0.0;
+        return RPM;
+    }
+    /**
+     * @return True if the target is within the turret's range of rotation, else false
+     */
+    public boolean inRotation() {
+        //returns whether or not the target is within the range that the turret can rotate to, used by driver
+        boolean inRotationRange=true;
+        return inRotationRange;
+    }
+    /**
+     * @return True if the target is within the horizontal distance from the target the shooter is capable of shooting to, else false
+     */
+    public boolean inRange() {
+        //returns whether or not the target is within the range that the shooter can shoot, used by driver
+        boolean inRange=true;
+        return inRange;
+    }
+    public void reverse() {
+        //reverses shooter motors
+        mFlywheelMasterMotor.setInverted(true);
+    }
+
+    public void reset() {
```

Comment:
@ethanrini I like this -- which brings to mind as to what is the *default* position for the turret angle and elevation?